core/embed/rust/src/strutil.rs

```diff
@@ -9,38 +9,16 @@ use crate::micropython::{buffer::StrBuffer, obj::Obj};
 #[cfg(feature = "translations")]
 use crate::translations::TR;
 
-/// Trait for slicing off string prefix by a specified number of bytes.
-/// See `StringType` for deeper explanation.
-pub trait SkipPrefix {
-    fn skip_prefix(&self, bytes: usize) -> Self;
-}
-
-// XXX only implemented in bootloader, as we don't want &str to satisfy
-// StringType in the main firmware. This is because we want to avoid duplication
-// of every StringType-parametrized component.
-#[cfg(feature = "bootloader")]
-impl SkipPrefix for &str {
-    fn skip_prefix(&self, chars: usize) -> Self {
-        &self[chars..]
-    }
-}
-
 /// Trait for internal representation of strings.
 /// Exists so that we can support `StrBuffer` as well as `&str` in the UI
 /// components. Implies the following operations:
 /// - dereference into a short-lived `&str` reference (AsRef<str>)
 /// - create a new string by skipping some number of bytes (SkipPrefix) - used
 ///   when rendering continuations of long strings
 /// - create a new string from a string literal (From<&'static str>)
-pub trait StringType:
-    AsRef<str> + From<&'static str> + Into<TString<'static>> + SkipPrefix
-{
-}
+pub trait StringType: AsRef<str> + From<&'static str> + Into<TString<'static>> {}
```

Comment:
Can we update the docs above? Namely, `SkipPrefix` part is obsolete and `Into<TString>` is not mentioned.

Reply:
c28dfe3db1bfdb2e1610e5875ffd9a0d9db0ea37

Reply:
:+1: for crutch